src/lib/Card/Card.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import { Card } from "./Card";
+import {
+  CardImageBorderRadius,
+  CardImageSize,
+  CardProps,
+  CardVariant,
+} from "./Card.types";
+
+const meta: Meta<typeof Card> = {
+  component: Card,
+  title: "Card",
+  tags: ["autodocs"],
+};
+export default meta;
+
+type Story = StoryObj<typeof Card>;
+
+export const SampleComponent: Story = (args: CardProps) => {
+  return <Card {...args} />;
+};
+
+SampleComponent.parameters = {
+  docs: {
+    canvas: { sourceState: "shown" },
+    story: { height: "500px" },
+  },
+};
+
+SampleComponent.args = {
+  children: <div style={{ width: "300px" }}>Team lead</div>,
+  title: "MangoBurger",
+  imageSize: CardImageSize.LARGE,
+  image: <img src="/images/default-avatar.png" />,
+  imageBorderRadius: CardImageBorderRadius.MEDIUM,
```

Comment:
Same as the comment above. I believe the documentation mentions `Circle` as the default, but here,`imageBorderRadius` is set to Medium.

Reply:
2 of the 3 cards in use have circle images (contest tile and reports tile) so I made that the default, but the team members are square. I'll change the storybook defaults to the component defaults.